### PR TITLE
Add mention of on Figma to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by kjellr**
The "How Can Designers Contribute?" section of `CONTRIBUTING.md` mentions an outdated Sketch file. Since the design team has largely switched to Figma, we should replace this outdated file with details on our team's Figma resources.

### CODE REVIEW

**Review by enriquesanchez — APPROVED** [review]
Looks good to me 👍

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Please see the [Developer Contributions section](/docs/contributors/develop.md)
...
+If you'd like to contribute to the design or front-end, feel free to contribute to tickets labelled [Needs Design](https://github.com/WordPress/gutenberg/issues?q=is%3Aissue+is%3Aopen+label%3A%22Needs+Design%22) or [Needs Design Feedback](https://github.com/WordPress/gutenberg/issues?q=is%3Aissue+is%3Aopen+label%3A"Needs+Design+Feedback%22). We could use your thoughtful replies, mockups, animatics, sketches, doodles. Proposed changes are best done as minimal and specific iterations on the work that precedes it so we can compare. The [WordPress.org Design team](http://make.wordpress.org/design/) uses [Figma](https://www.figma.com/). If you'd like to contribute, consider [joining the The WordPress.org Figma team](https://make.wordpress.org/design/2018/11/19/figma-for-wordpress/). The Figma team has access to a helpful [library of components](https://www.figma.com/file/ZtN5xslEVYgzU7Dd5CxgGZwq/WordPress-Components?node-id=0%3A1) used in WordPress.
```
Comment (sarahmonster) [inline CONTRIBUTING.md]: A few small suggested changes here: 

- "WordPress.org Design team" → "WordPress design team"
- "uses Figma" → "uses Figma to more easily collaborate and share work" (or similar)
- "consider joining The WordPress.org Figma team" → "ping in Slack and the design team will set you up with a free Figma account" (the process is a bit different and basically you need to ping an admin)

**Review by sarahmonster — APPROVED** [review]
Works by me! I've left a comment but it's all quite minor, so I'd say we can go ahead and ship this as is.

We might want to update this once our Figma libraries are a bit more fleshed out to include all the different libraries, but since they're still in flux pointing to the main one is a good start and sufficient for now.

File: CONTRIBUTING.md (hunk 1 above)
Comment (kjellr) [reply CONTRIBUTING.md]: Thanks @sarahmonster! Updated in 5ad4184.

Comment (sarahmonster) [reply CONTRIBUTING.md]: 💯 that reads a million times better. 🙌 🚢

**Pull request merged**